components/test_storage/src/sync_storage.rs

```diff
@@ -165,12 +164,15 @@ impl<E: Engine> SyncTestStorage<E> {
         primary: Vec<u8>,
         start_ts: impl Into<TimeStamp>,
     ) -> Result<Vec<Result<()>>> {
-        wait_op!(|cb| self.store.async_prewrite(
+        wait_op!(|cb| self.store.prewrite(
             ctx,
             mutations,
             primary,
             start_ts.into(),
-            Options::default(),
+            0,
```

Comment:
I will remove some in a follow-up PR, this PR is an intermediate stage. We can also squash some arguments together later.

The trouble with `Options` is that it is used by different commands, but different commands use different options, so that makes things more loosely typed. That leads to mistakes, for example, `txn_size` is read by `acquire_pessimistic_lock` but cannot be set for that command (except in tests).